src/main/resources/application.yaml

```diff
@@ -24,7 +24,9 @@ idam:
     secret: ${DISPOSER_IDAM_USER_CLIENT_SECRET:idam-client-secret}
     redirect_uri: ${DISPOSER_IDAM_USER_CLIENT_REDIRECT_URL:https://disposer-idam-user/oauth2/callback}
 
-stale-users.batch.size: ${DISPOSER_IDAM_USER_BATCH_SIZE:100}
+stale-users:
+  batch.size: ${DISPOSER_IDAM_USER_BATCH_SIZE:100}
+  requests.limit: ${DISPOSER_IDAM_USER_REQUESTS_LIMIT:100}
```

Comment:
Do you think this default is good enough given that there might be millions of accounts to delete? I understand that this can also come from environment, but if we forget to set or pass..